test/unittests/test_analyse_xml_data/test_construct_and_analyze_graph.py

```diff
@@ -214,14 +214,14 @@ def test_make_graph() -> None:
     xml = XMLLink("a", {"b", "c"})
     xml_links = [xml]
     all_ids = ["a", "b", "c"]
-    graph, node_index_lookup, edges, node_indices = make_graph(resptr_links, xml_links, all_ids)
+    graph, rustworkx_index_to_id, edges = make_graph(resptr_links, xml_links, all_ids)
```

Comment:
again, I would not put "rustworkx" in any variable names

Reply:
Ok, I will try to improve that.